absl/strings/escaping.cc

```diff
@@ -41,6 +41,11 @@ constexpr bool kUnescapeNulls = false;
 
 inline bool is_octal_digit(char c) { return ('0' <= c) && (c <= '7'); }
 
+inline bool is_octal_number(int num) {
```

Comment:
It is meaningless to ask if an integer is an octal number. All integers have an octal representation.

This is also a change to a `.cc` file that isn't exposed in any header, so this new function wouldn't be usable anyway.